contents/pods-run-script.py

```diff
@@ -62,6 +63,11 @@ def main():
         exit(1)
 
     script = os.environ.get('RD_CONFIG_SCRIPT')
+
+    # Python 3 expects bytes string to transfer the data.
+    if PY == 3:
+        script = script.encode('utf-8')
+
```

Comment:
Additionally, Line number #91 is missing in our selection of commit for this plugin